docs/tools/glaze/example.md

```diff
@@ -0,0 +1,271 @@
+# Glaze example
+
+## 1. Prerequisites
+
+In this example, we'll create a DataModel for a simple text note that can be associated to a DID, and an associated script to interact with this model.
+
+First, we will need to install the following runtime packages:
+
+```sh
+npm install @ceramicnetwork/http-client @glazed/datamodel @glazed/did-datastore dids key-did-provider-ed25519 key-did-resolver uint8arrays
+```
+
+The following steps will present two ways of going through the development flow: creating a custom script (recommended option for complex models) or using the Glaze CLI (simpler option for basic models):
+
+=== "Using a script"
+
+    ```sh
+    npm install --dev @glazed/devtools
+    ```
+
+=== "Using the CLI"
+
+    ```sh
+    npm install --global @glazed/cli
+    ```
+
+## 2. Local setup
+
+=== "Script"
+
+    In order to create a DataModel, we need a Ceramic instance with an authenticated DID.
+    DataModels only support DIDs using the `did:key` method, so we'll `key-did-provider-ed25519` in this example.
+
+    Let's create a `create-model.mjs` file with the following contents:
+
+    ```js
+    import { writeFile } from 'node:fs/promises'
+    import Ceramic from '@ceramicnetwork/http-client'
+    import { ModelManager } from '@glazed/devtools'
+    import { DID } from 'dids'
+    import { Ed25519Provider } from 'key-did-provider-ed25519'
+    import KeyResolver from 'key-did-resolver'
+    import { fromString } from 'uint8arrays'
+
+    // The key must be provided as an environment variable
+    const key = fromString(process.env.DID_KEY, 'base16')
+    // Create and authenticate the DID
+    const did = new DID({
+      provider: new Ed25519Provider(key),
+      resolver: KeyResolver.default.getResolver(),
+    })
+    await did.authenticate()
+
+    // Connect to the local Ceramic node
+    const ceramic = new Ceramic.default('http://localhost:7007')
```

Comment:
I guess this can be updated with the new exports?

Reply:
Yes good catch, thanks!